Library/ViewModels/PledgePaymentMethodsAndSelectionData.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+public struct PledgePaymentMethodsAndSelectionData {
+  public let paymentMethodsCellData: [PledgePaymentMethodCellData]
+  public let paymentSheetPaymentMethodsCellData: [PaymentSheetPaymentMethodCellData]
+  public let selectedPaymentMethod: PaymentSourceSelected?
+  public let shouldReload: Bool
+  public let isLoading: Bool
+
+  public func updateWithPaymentMethodsData(_ newPaymentMethodsData: [PledgePaymentMethodCellData])
```

Comment:
These methods replace the use of the Prelude templating for this tuple - they return new copies of the object, mutating the correct variable.

Reply:
There is probably a smarter and less verbose way to do all of these 🤷‍♀️